conda-recipe/run_test.bat

```diff
@@ -28,6 +28,8 @@ if "%PYTHON%"=="python" (
     set NO_DIST=1
 )
 
+IF DEFINED COVERAGE_RCFILE (set COV_ARGS=--cov=onedal --cov=sklearnex --cov-config=%COVERAGE_RCFILE% --cov-append --cov-report=) else (set COV_ARGS=)
```

Comment:
done, also changed ``` python``` above it

Reply:
I think the `defined` call requires having the variable name without the percent signs, otherwise it will apply substitutions on it and the statement might not work. Also looks like the double quotes were lost.

Reply:
Changes as requested made.